aquadoggo_cli/src/main.rs

```diff
@@ -54,41 +55,41 @@ struct Opt {
     /// .. adds the authors:
     /// - "123abc" with log_ids 1, 2, 345
     /// - "456def" with log_ids 6 7
-    #[structopt(short = "A", parse(try_from_str = parse_key_val), number_of_values = 1)]
+    #[arg(short = 'A', value_parser = parse_key_val::<String, u64>, number_of_values = 1, verbatim_doc_comment)]
```

Comment:
`verbatim_doc_comment` is needed to ensure that the help message for `public_keys_to_replicate` appears as it does in the doc comment (ie. with newlines), otherwise the newlines are ignored in the output.

I wasn't convinced that `parse_key_val::<String, u64>` was going to work as intended (parsing a `Vec<u64>` and not just a singe `u64`) so I quickly added a debug statement to `replication/service.rs` to log the value:

`RUST_LOG=aquadoggo=debug cargo run -- -A 7cf4f58a2d89e93313f2de99604a814ecea9800cf217b140e9c3a7ba59a5d982="1 2 3"`

`[2023-03-15T07:24:43Z DEBUG aquadoggo::replication::service] public_keys_to_replicate: [PublicKeyToReplicate(PublicKey(PublicKey(CompressedEdwardsY: [124, 244, 245, 138, 45, 137, 233, 51, 19, 242, 222, 153, 96, 74, 129, 78, 206, 169, 128, 12, 242, 23, 177, 64, 233, 195, 167, 186, 89, 165, 217, 130]), EdwardsPoint{
    	X: FieldElement51([1434015213083887, 807896449253015, 2050930520079949, 2023919544359233, 1458722956326961]),
    	Y: FieldElement51([432303674619004, 1147047759734397, 565867196319017, 550444494223622, 50140884478588]),
    	Z: FieldElement51([1, 0, 0, 0, 0]),
    	T: FieldElement51([325029136006990, 629106044855861, 2077527849940718, 1535784221899331, 111353764278980])
    })), [LogId(1), LogId(2), LogId(3)])]`

As we can see from `[LogId(1), LogId(2), LogId(3)]`, it appears to work as intended :)